### PR TITLE
Copy editing

### PR DESCRIPTION
**Pull request by alexryangregor**
Since this is the first page I've completed, I am including extensive comments for context.

As per Erik's request, I have prioritized grammar, while also ensuring that the content makes sense. I have improved flow, clarity, consistency and quality of the language and readability, when necessary.

I have been reviewing other SEPAL publications to ensure consistency across publications; I have also been reviewing other documentation from GitHub, Microsoft, and Apple to improve flow, clarity, and quality of the language and readability in the SEPAL documentation, as I've seen in the documentation from these organizations (mostly using them as model texts for the presentation of SEPAL content to improve user experience and increase user engagement).

A few notes (moving from the top of the page to the bottom):

-I changed the subheading, "Introduction" to the subheading above mostly because I was confused that there were two "Introduction[s]": one on this page, and another in the "Getting started with SEPAL" section ("Introduction to SEPAL" in the left panel). I found a brief summary of the page's content as a subheading to be common on other organizations' "Getting Started" sections, so I applied this approach here.
-I explicitly stated all of the features of the "Getting started with SEPAL" section, rather than only two ("Register to SEPAL" and "Exchange files with SEPAL"), as I think it's important to include all of them. Each title should be a link.
-I applied language that I found in other organizations' documentation throughout (e.g. changing "you will" to "you can"), with an aim to improve user experience and increase user engagement. 
-I diversified word choice (e.g. finding synonyms for "use").
-I broke up blocks of text into separate paragraphs to improve readability. I noticed, in other organizations' documentation, that lists (numbered and bulleted) seem to be utilized more than blocks of text. I suggest considering using lists when possible (e.g. in the second body paragraph above that begins with "This guide includes...").
-I noticed in other organizations' documentation that "Ask the community", "Contact support", and "Make a contribution" are common buttons and/or links at the bottom of every page. I recommend considering doing this, so that users can quickly and easily take action when necessary (this will also improve user experience and increase user engagement, as well).
-I changed the the word "button" to "link" in the last paragraph, as "Edit this page" doesn't display as a button in the right panel.

### CODE REVIEW

File: docs/source/setup/index.rst [hunk 1]
```diff
@@ -1,16 +1,19 @@
 Getting started with SEPAL
 ==========================
 
-Introduction
+Set up an account and learn how to access the SEPAL interface
 ------------
```
Comment (12rambau) [inline docs/source/setup/index.rst:5]:
```suggestion
---------------------------------------------------------------
```

this is something you cannot see but when the documentation is built I see some parsing errors. 
Lines under the title need to be the same length

File: docs/source/setup/index.rst [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
 
-In this section of our documentation you will find the key requirements to use the platform. We'll start from :doc:`register` to the more advanced :doc:`filezilla`. use the left panel to navigate in this section of the documentation. 
-If you don't find what you need please let us know using the `GitHub issue tracker <https://github.com/openforis/sepal-doc/issues/new/choose>`_ of this very website.
+In this section of SEPAL documentation, you can learn about the key requirements needed to use the platform.
 
-We will be happy to receive contributions from people that use this documentation, so if you find a typo, don't hesitate to use the :btn:`<fas fa-pencil-alt> edit this page` button on the right side of every documentation page to help us improve the way we present the platform!
+This guide includes an Introduction to the SEPAL interface, as well as instructions that explain how you can :doc:`register`, Use Google Earth Engine (GEE) with SEPAL, :doc:`filezilla`, Use Norway’s International Climate & Forests Initiative (NICFI) - Planet Lab data, and Reset your password.
```
Comment (12rambau) [inline docs/source/setup/index.rst:9]:
```suggestion
This guide includes:
-   An Introduction to the SEPAL interface
-   Instructions that explain how you can :doc:`register`
-   Usage instructions for  Google Earth Engine (GEE) with SEPAL
-   Use of :doc:`filezilla` to download files
-   Use Norway’s International Climate & Forests Initiative (NICFI) - Planet Lab data
-   How to reset your password.
```

this is an example of how you can create a bullet point list. "-" and then 3 spaces, you can accept this suggestion and then modify the n text again if needed.

**Comment by 12rambau** [comment]
> -I noticed in other organizations' documentation that "Ask the community", "Contact support", and "Make a contribution" are common buttons and/or links at the bottom of every page. I recommend considering doing this, so that users can quickly and easily take action when necessary (this will also improve user experience and increase user engagement, as well).

Note that this is a new component that I'm working on in the `pydata-sphinx-theme`, it should be released in v0.11. I'm moving this suggestion to issues so that I remember when it will be available

**Review by 12rambau — CHANGES_REQUESTED** [review]
thanks for your PR ! 

I made some comments and added @eriklindquist to the reviewer list. 
Let me know by teams if you don't remember how to accept suggestions and work on top of them

**Comment by alexryangregor** [comment]
> 

Thanks Pierrick! More soon.

In the meantime, could you provide me with access to the "Edit this page" feature for [this page](https://docs.sepal.io/en/latest/modules/index.html)?

It's on the list, but I'm receiving the following 404 message:

image

Thanks,
Alex

**Review by 12rambau — APPROVED** [review]

**Review by eriklindquist — APPROVED** [review]

**Pull request merged**